src/main/java/edu/kis/powp/jobs2d/command/gui/CommandManagerWindow.java

```diff
@@ -21,7 +21,7 @@
 	private List<Subscriber> observerList;
 	private boolean observersDeleted = false;
 	private final CommandManager commandManager;
-
+	private DriverCommand currentCommand = null;
```

Comment:
a new unused field?

Reply:
fixed in next commit, forgot to remove it in first place